crates/bevy_openxr/src/openxr/spaces.rs

```diff
@@ -140,7 +140,7 @@ impl OxrSpaceVelocityFlags {
 }
 
 #[allow(clippy::type_complexity)]
-fn update_space_transforms(
+pub fn update_space_transforms(
```

Comment:
this should not be made public, there is a system set to schedule against this system

Reply:
which system set is that?

Reply:
`OxrSpaceSyncSet`